### DISCUSSION TITLE
ERROR: Zero or negative volume for 3D-Element

### DISCUSSION THREAD

**Discussion by Calvinsgithub** [OpenRadioss/OpenRadioss#3247] [Q&A]
Hi, 
I was trying to make a milling model, but ran into this problem at the beginning of the simulation: ERROR: Zero or negative volume for 3D-Element. as shown in the image. Attached is my code file and I would appreciate any guidance from you.

![{P`)ME5((CK3C}YD A WL)H](https://github.com/user-attachments/assets/7a65793a-0edb-4ffb-af9c-f2c4bae5ba9c)
[test.zip](https://github.com/user-attachments/files/18205183/test.zip)

**Comment by PaulAltair** [comment]
hi, there is nothing attached, the link is just a link back to this ticket

**Reply by Calvinsgithub** [reply]
I am very glad to receive your reply Paul, I have updated the file and hope to receive your suggestions.

**Reply by Taylor1987** [reply]
I was also frequently encounter the same issue, hope I can also give some suggestions

**Comment by PaulAltair** [comment]
There are many issues with what you shared, several of which may have caused the iniital solid element failure.

Both your modulus and density values were too large (by a factor of a billion each), your timestep was adding a lot of added mass, your constraints were not applied on anything (sets were empty), you had only initial velocities assigned (no imposed velocities) the initial velocity on the tool was only on the rigid nodes. The Type 7 contact was only one way (tool - test piece) and you had left no clearance between the parts (Type 7 must have an initial separation) You had defined a param for the rotational velocity, but hadn't used it.

I have made a lot of assumptions as to what you were aiming for in this attached version so it probably still isn't what you want (I corrected densities and moduli, redefined contact with a symmetrical copy, moved the test part slightly, added imposed velocities and used your parameter for rotational velocity)

But this run at least initialises and starts running, you ought to be able to modify further from here now it is running to get closer to what you need:  I'm taking a couple of weeks off now for the holidays so apologies, it may take a little while if you have follow up questions!

[PS_002.zip](https://github.com/user-attachments/files/18223855/PS_002.zip)

**Reply by Calvinsgithub** [reply]
Enjoy your vacation and thank you for your quick reply. Paul, when I use the stater_win64.exe -i 0000_rad-np-i command to check the solution file, the new problem as shown in the following figure occurs. May I ask what causes it? How can I solve it? Looking forward to your reply.
![V$V_A9`945U~`6VY24R~7LX](https://github.com/user-attachments/assets/cf9c4e81-e79b-4800-8db8-a09404708df6)
